GameDataParser/Parsers/ExpParser.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace GameDataParser.Parsers
 {
-    class ExpTableParser : Exporter<List<ExpMetadata>>
+    class ExpParser : Exporter<List<ExpMetadata>>
```

Comment:
Why is this changing again? From `ExpParser` to`ExpTableParser` then again `ExpParser`?